tests/testapp/settings.py

```diff
@@ -58,3 +58,21 @@
         },
     },
 ]
+
+LOGGING = {
```

Comment:
I don't see great value having the logger enabled in tests where everything is under control (i.e. we're not debugging) so I'd rather we omitted it because it messes with test output.